ChangeLog.d/remove_obsolete_tls_features.txt

```diff
@@ -1,7 +1,9 @@
 API changes
-   * Drop support for parsing SSLv2 ClientHello (MBEDTLS_SSL_SRV_SUPPORT_SSLV2_CLIENT_HELLO).
+   * Drop support for parsing SSLv2 ClientHello
+     (MBEDTLS_SSL_SRV_SUPPORT_SSLV2_CLIENT_HELLO).
    * Drop support for SSLv3 (MBEDTLS_SSL_PROTO_SSL3).
-   * Drop support for compatibility with our own previous buggy implementation of truncated HMAC (MBEDTLS_SSL_TRUNCATED_HMAC_COMPAT).
+   * Drop support for compatibility with our own previous buggy
+     implementationof truncated HMAC (MBEDTLS_SSL_TRUNCATED_HMAC_COMPAT).
```

Comment:
Typo: "implementation of" (missing space).

Reply:
omg. Sorry about that. Amended the commit

Reply:
No worries, happens all the time.